src/utils/venue.ts

```diff
@@ -21,8 +21,8 @@ export const canHavePlacement = (venue: Venue): boolean =>
   PLAYA_TEMPLATES.includes(venue.template);
 
 export const peopleByLastSeenIn = (
-  users: readonly WithId<User>[] | undefined,
-  venueName: string
+  venueName: string,
+  users?: readonly WithId<User>[]
```

Comment:
These are function parameters, not keys in a 'props' object, so changing their order like this will change their purpose. Have we checked/updated all places this function is used to ensure it is correctly updated?

What was the reasoning behind changing the parameter order here? I'm guessing the linting said you can't have an optional parameter first? Which if so, is probably why it was written the way it was previously, even though that is kind of incorrect too.. Based on usages, is there any reason we ever need to accept `undefined` here? From the name of the function it sounds like it would be sort of 'purposeless' when `undefined`, so maybe we can just push the defaulting with `?? []` to the call sites where it might be `undefined`?

Reply:
I didn't specifically look into the logic of this fn, but I made sure to change it accordingly in places where it is used.
Linter was screaming about this, yes.

Reply:
Probably, it is good to make an issue about that in a different PR

Reply:
So long as you made sure to update it's usage correctly in all the places, that satisfies my concerns for this PR.